Druid-Tests/DruidTestRTLCompiler.class.st

```diff
@@ -2599,9 +2576,9 @@ DruidTestRTLCompiler >> gen_primitiveModByConstant [
 	self
 		DivR: ClassReg
 		R: TempReg
-		Quo: ClassReg
-		Rem: TempReg.
-	self MoveR: TempReg R: ReceiverResultReg.
+		Quo: TempReg
+		Rem: ClassReg.
```

Comment:
It would be nice to remove the division here if possible.
We know that TempReg contains the format, so it will be 5 bits long.
Also we have all the flow information to kind of reduce this to a couple of choices.
Can then we add some flow info, then see if we can strenght reduce this to a shift and a couple of subtractions?